specs/spec-1-kudos.md

```diff
@@ -1,106 +1,310 @@
+## Changelog
+
+<!--
+Points for reviewers to consider:
+
+- Be pragmatic: Is there enough information to begin coding? Practically, implementation will reveal the unforeseeable. This doc can evolve to record those issues/resolutions.
+- Wide angle view: Do you know of overlapping projects? Are there OKRs or other non-technical forces that would push/pull priority?
+- Optimization: Describe alternate approaches and why they are better. Use Caveats section for approaches under consideration, even if they are ultimately abandoned. 
+
+--> 
+
+**Authors & Reviewers:** @starpause, @robert-zaremba, @htafolla
+*Please insert your name here if you do review the document.*
+
+| Ver. | Date       | Author          | Changes Made                             |
+| ------- | ---------- | --------------- | ---------------------------------------- |
+| 0.1.0   | 2023-04-21 | @starpause      | Initial draft of the document            |
+| 0.2.0   | 2023-04-23 | @starpause      | Merged prior writting by @robert-zaremba from his [Github version](https://github.com/near-ndc/gwg/pull/1)                   |
+| 0.3.0   | 2023-04-25 | @starpause      | Addressed review comments, formated sections to more closely match [template](https://github.com/near-ndc/gwg/blob/main/TEMPLATE-SPEC.md)                                                                            |
+
+# Open Questions
+
+<!-- Topics not covered in this document that need to be addressed in a meeting or async -->
+
+**Q:** Should use Social DB for upvotes?
+
+**A:**
+
+**Q:** Should we issue SBT when someone gives a Kudo (Con: Feels spammy, costs more)? Or when someone claims their SBT after reaching a # of upvotes (Con: Less SBT will be issued)?
+
+**A:**
+
+**Q:** What happens when a Kudo expires? 
+
+**A:**
+
+**Q:** Is commenting limited to verified humans?
+
+**A:**
+
+**Q:** Who would "follow a kudo" and why? What notifications would such a follow recieve? 
 
-## Context
+**A:**
 
-The goal of this widget is to foster and improve recognition and celebrations of individuals, nodes, projects, workgroups or collectives for their contribution to NEAR.
-This will server as a fundamental MVP for Reputation and Badges and create a general widget that can be forked and used for many use cases
+**Q:** What threshold can we define for when an audit is appropriate? 
 
-Other potential names for this widget:
+**A:**
+
+# Overview
+
+<!-- One page high-level overview; put details in the specification section and background in the previous section. Should be understandable by a new engineer or reader who isn't already working on the project. -->
+
+## Challenge
+
+<!-- List the challenge(s) being solved by this widget. What is the problem? -->
+
+The specific purpose of the Kudos widget is to crowd source reputation through gamification. Kudos needs to foster recognition and celebrations of individuals, nodes, projects, workgroups or collectives for their contribution to NEAR. 
```

Comment:
can a project or a node make an upvote or make a kudo? (if we stick to i-am-human, then probably not)